com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeSubdivide.compute

```diff
@@ -0,0 +1,261 @@
+// Each #kernel tells which function to compile; you can have many kernels
+#pragma kernel Clear
+#pragma kernel ClearBuffer
+#pragma kernel FillUVMap
+#pragma kernel JumpFlooding
+#pragma kernel FinalPass
+#pragma kernel VoxelizeProbeVolumeData 
+#pragma kernel Subdivide
+
+#pragma enable_d3d11_debug_symbols
+
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl"
+#include "Packages/com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbePlacement.cs.hlsl"
+
+Texture3D _Input;
+Texture3D _ProbeVolumeData;
+RWTexture3D<float4> _Output;
+RWTexture3D<float4> _FinalOutput;
+
+AppendStructuredBuffer<float3> _Bricks;
+RWStructuredBuffer<float3> _BricksToClear;
+
+StructuredBuffer<GPUProbeVolumeOBB> _ProbeVolumes;
+
+float3 _Size;
```

Comment:
Generally I see a bunch of those are float3 and can be float (float3 * float is well defined in HLSL)

Also prefer to pass float4 since these are going to go into a cbuffer and padded singularly, what I usually do is send a bunch of 

float4 SDFParams1...N

and then do defines with proper naming in shader of the granularity you need so for example 

#define  _Offset SDFParams1.w